src/mpv.rs

```diff
@@ -100,6 +110,18 @@ pub fn simple_example(
                     Ok(Message::Backward) => mpv
                         .command(&["seek", "-5"])
                         .expect("Error setting MPV property"),
+                        Ok(Message::SpeedUp) => {
+                            let speed: f64 = mpv.get_property("speed").unwrap();
+                            let new_speed: f64 = (speed + 0.25).min(5.0);
```

Comment:
I'm no rust dev, but should `min` not be `max` here?

Reply:
I think he is trying to set a max allowed value. 
He check min(speed + 0.25 vs 5.0), so the max alowed value will be 5.0.  

For example if speed is 5.0 and u try to speed up you will stand on 5.0 due to min check.

On the other hand... I think SpeedDown method is wrong, it should be max.